recipes/example/meta.yaml

```diff
@@ -1,81 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Do not delete the `example/meta.yaml` file.

Reply:
Thanks! I have brought it back.